pkg/ir/derivation_test.go

```diff
@@ -242,27 +242,30 @@ func TestFeatureDerivation(t *testing.T) {
 	fc1 = trainStmt.Features["feature_columns"][0]
 	nc, ok = fc1.(*NumericColumn)
 	a.True(ok)
+	a.Equal("c1", nc.FieldDesc.Name)
 
 	fc2 = trainStmt.Features["feature_columns"][1]
-	nc, ok = fc2.(*NumericColumn)
+	cc, ok := fc2.(*CrossColumn)
```

Comment:
Feature column order should be the same as the `COLUMN` clause defined. Since the test case column clause is: `COLUMN c1, c2, CROSS([c1, c2], 256)`

Reply:
Done.